code/workspaces/web-app/src/components/assetRepo/AssetCard.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+  row: {
+    display: 'flex',
+  },
+  key: {
+    fontWeight: 'lighter',
+    width: theme.spacing(20),
+    marginBottom: theme.spacing(1),
+  },
+  value: {
+    marginBottom: theme.spacing(1),
+  },
+  list: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+}));
+
+function AssetCardKeyValue({ rowKey, rowValue }) {
+  const classes = useStyles();
+  return (
+    <div className={classes.row}>
+      <div className={classes.key}>{rowKey}</div>
+      <div className={classes.value}>{rowValue}</div>
+    </div>
+  );
+}
+
+function AssetCardHyperlink({ rowKey, rowValue }) {
+  const classes = useStyles();
+  return (
+    <div className={classes.row}>
+      <div className={classes.key}>{rowKey}</div>
+      <div className={classes.value}><a href={rowValue} target="_blank" rel="noopener noreferrer">{rowValue}</a></div>
+    </div>
+  );
+}
+
+function AssetCardKeyList({ rowKey, rowList }) {
+  const classes = useStyles();
+  return (
+    <div className={classes.row}>
+      <div className={classes.key}>{rowKey}</div>
+      <div className={classes.list}>
+        {rowList && rowList.map(value => (<div key={value} className={classes.value}>{value}</div>))}
+      </div>
+    </div>
+  );
+}
+
+function AssetCard({ asset }) {
+  const visibility = {
+    PUBLIC: 'Public',
+    BY_PROJECT: 'By project',
+  };
```

Comment:
Is this something to be updated once the approach to using config maps in the UI has been formalised and implemented for the other config maps (i.e. the outstanding technical debt story for this has been completed)?

Reply:
Yes, I've made a note in that story.